enviro/plot.py

```diff
@@ -418,54 +424,215 @@ def define_header_and_footer(canvas, doc):
     canvas.restoreState()
 
 
-def plot_pdf(matrix, user, method_label, probabilistic_model, var_names, var_symbols, method):
+def create_latex_report(matrix, user, method_label, probabilistic_model, var_names, var_symbols, method):
     """
-    The function generates a pdf. The pdf includes an image of the contour and a table with the data points.
-    :param matrix:      data points supported by compute. 
-    :param user:        who starts the order. 
-    :param method_label:      e.g. "T = 25 years, IFORM"
-    :param probabilistic_model:       probabilistic model object.
-    :param var_names:   names of the variables.
-    :param var_symbols: symbols of the variables of the probabilistic model
-    :return:            the path to the user related pdf.
+    Creates a latex-based pdf report describing the performed environmental contour calculation.
+
+    Makes use of the 'latex_report.tex' template where the document class and packages are defined.
+
+    Parameters
+    ----------
+    matrix : n-dimensional matrix
+        The coordinates of the environmental contour.
+        The format is defined by compute_interface.iform()
+
+    user : Django User instance
```

Comment:
user.models.User

Reply:
I think this should be "user : django.contrib.auth.models.User"